### DISCUSSION TITLE
No configuration options in latest version via web?

### DISCUSSION THREAD

**Discussion by teredactle** [gtxaspec/wz_mini_hacks#645] [Q&A] [answered]
Trying to get another go to this, loaded the latest version and have web enabled. When I go to the IP all I see is:

Wyze Mini Web Sever Menu
[Go2RTC Server]

Nothing else. Before there were other links including one where a user could edit the config file and then reboot.
What happened?

**Comment by virmaior** [comment, marked as answer]
you need to set an option wz_mini.conf `WEB_SERVER_OPTIONS`

Aren't you seeing a message like
```
wz_mini.conf is missing the line WEB_SERVER_OPTIONS 
to populate it with full capabilities:<pre>WEB_SERVER_OPTIONS="cam config car jpeg multicam diag status"</pre>
```
?

(code behind that is at https://github.com/gtxaspec/wz_mini_hacks/blob/master/SD_ROOT/wz_mini/etc/network.d/S10httpd )

**Reply by teredactle** [reply]
I didn't see any message, but not sure where I would see it anyway...
 Thank you, I will add this,  not sure if in the past this was in the conf by default, I don't recall adding it so totally missed it in the wiki/config file write up...

**Reply by prsk8er735** [reply]
I would like to put in question why this line is not default in the .conf file to begin with? If this is a requirement for the webserver to be enabled, it should be in the conf file by default. Then when web_server= true, it would read in the web_server_options line. Seems odd to obfuscate it like this if its a hard requirement to enable the web server.

**Reply by virmaior** [reply]
basically largely my fault not understanding how to add that to the config stuff.

I contributed a lot of the web code but wasn't sufficiently aware of the other working parts to know how to edit it ...

**Comment by virmaior** [comment]
and now I see my error  (left off the `>> $target `) on those two lines...
https://github.com/gtxaspec/wz_mini_hacks/pull/646

**Comment by fotingo** [comment]
Hello, 
I need help viewing the camera using a web browser. I am too getting that message
wz_mini.conf is missing the line WEB_SERVER_OPTIONS

but I don't know what I need to do.
Thanks.

**Reply by virmaior** [reply]
edit `/opt/wz_mini/wz_mini.conf`
add 
```
WEB_SERVER_OPTIONS="cam config jpeg car status"
```

and you'd add access to every function.

**Reply by fotingo** [reply]
Thank you I got it!
I notice the Wyze issue on RTSP is that every few minutes the video disconnects and reconnects. I thought by installing this it would fix that issue. This however only happens when disabling internet access to use on LAN only. 

If I enable internet access, this issue goes away. But this has been this way since I can remember with Wyze which is why I stopped using them. Some say it's because the camera tries to "phone home" every few minutes, which I don't like.
